src/pages/starter-kit/checkout/connect.md

```diff
@@ -18,18 +18,26 @@ To use the Adobe Commerce HTTP Client, update `COMMERCE_BASE_URL=<commerce_insta
 
 Depending on your Adobe Commerce setup, there are two options to authenticate and communicate with App Builder:
 
-- [Configure Adobe Identity Management Service (IMS)](#configure-adobe-identity-management-service-ims)
-
 - [Configure Commerce Integration](#create-a-commerce-integration)
 
-If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, than client instantiation will fail.
+- [Configure Adobe Identity Management Service (IMS)](#configure-ims)
+
+If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, then client instantiation will fail.
+
+### Configure IMS
 
-### Configure Adobe Identity Management Service (IMS)
+The following sections describe how to configure IMS for Commerce and App Builder authentication.
 
-IMS authentication requires the following setup:
+#### Configure IMS for Commerce
```

Comment:
Instead of referring to this point as IMS configuration, wouldn't be better to just point to Admin SDK configuration if you plan to use it (https://developer.adobe.com/commerce/extensibility/admin-ui-sdk/installation/)

Reply:
@obarcelonap, I've updated the PR to make the reference to Admin UI SDK a note.